network_test.go

```diff
@@ -322,3 +324,18 @@ func TestListRoutes(t *testing.T) {
 	assert.True(reflect.DeepEqual(results.Routes[1], inputRoutesSimple[1]),
 		"Route listed didn't match: got %+v, expecting %+v", results.Routes[1], inputRoutesSimple[1])
 }
+
+func TestSetupDNS(t *testing.T) {
+	tmpfile, err := ioutil.TempFile("", "resolv.conf")
+	assert.Nil(t, err)
+	guestDNSFile = tmpfile.Name()
+	defer os.RemoveAll(guestDNSFile)
+
+	dns := []string{
+		"nameserver 8.8.8.8",
+		"nameserver 8.8.4.4",
+	}
+
+	err = setupDNS(dns)
+	assert.Nil(t, err)
```

Comment:
Test looks good. Would be better if you check the contents of the file are actually what you expect to see.